### DISCUSSION TITLE
Labeling Hong Kong, Macau and Singapore for AdGuard Chinese

### DISCUSSION THREAD

**Discussion by cherinyy** [uBlockOrigin/uBlock-issues#2753] [Ideas] [answered]
Currently uBO only labels the Chinese mainland and Taiwan for AdGuard Chinese, perhaps consider adding Hong Kong, Macau and Singapore, where Mandarin is also one of the official languages.

**Comment by ghost** [comment, marked as answer]
1. Hong Kong and Macau are Chinese cities
2. [There are more Chinese speakers in Malaysia](https://en.wikipedia.org/wiki/Languages_of_Malaysia#Malaysian_Chinese_languages) than the total population of Singapore
3. The purpose of the flag icons is just to help people find lists that include the languages they use online faster. Languages such as Arabic, French, Portuguese, and Spanish are spoken in several countries, and I believe you'd agree that it'd be insane to add all of their flags to the list.
